src/api/interfaces/generic_storage.rs

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
I would just call it `storage` the name already implies generic.

Reply:
Renamed to `state`

Reply:
Thanks!